test/Querier.test.ts

```diff
@@ -133,38 +135,34 @@ describe('Querier', () => {
 
     it('allows subscription cancelation', () => {
       const querier = new Querier();
-      const unsubscribe = querier.subscribe('key', jest.fn());
+      const subscribe = querier.subscribe('key', jest.fn());
```

Comment:
`querier.subscribe` returns function that unsubscribes from changes, that's why the verbose name here

Reply:
Ok!